src/definitions.ts

```diff
@@ -84,4 +84,10 @@ export interface RecordExceptionOptions {
    * Only available for iOS.
    */
   domain?: string;
+  /**
+   * A stacktrace generated by stacktrace.js
```

Comment:
It should be noted that `code` and `domain` are ignored when `stacktrace` is provided.

Reply:
I agree, I also added a hint to `code` and `domain`.